app/mmstyle.h

```diff
@@ -54,7 +54,7 @@ class MMStyle: public QObject
     // Colors - secondary palette
     Q_PROPERTY( QColor lightGreenColor READ lightGreenColor CONSTANT )
     Q_PROPERTY( QColor mediumGreenColor READ mediumGreenColor CONSTANT )
-    Q_PROPERTY( QColor greyColor READ greyColor CONSTANT )
```

Comment:
Heh, I changed it to `grey` in the previous PR. Looks like a war between US and UK english https://thisonevsthatone.com/grey-vs-gray/ :D

Reply:
I'd prefer to stick to `grey` though, it is how the design system names it. It makes it easier then to apply it